lib/compat/wordpress-6.1/block-template-utils.php

```diff
@@ -248,6 +248,115 @@ function gutenberg_get_block_template( $id, $template_type = 'wp_template' ) {
 	return apply_filters( 'get_block_template', $block_template, $id, $template_type );
 }
 
+/**
+ * Builds the title and description of a post specific template based on the underlying post referenced.
+ *
+ * @access private
+ * @internal
+ *
+ * @param string                   $post_type Post type e.g:page, post, product.
+ * @param string                   $slug      Slug of the post e.g:a-story-about-shoes.
+ * @param Gutenberg_Block_Template $template  Template whose description and title are going to be computed.
+ */
+function _gutenberg_build_title_and_description_for_single_post_type_block_template( $post_type, $slug, &$template ) {
+	$post_type_object = get_post_type_object( $post_type );
+
+	$posts = get_posts(
+		array(
+			'name'      => $slug,
+			'post_type' => $post_type,
+		)
+	);
+	if ( empty( $posts ) ) {
```

Comment:
If we don't find the post or taxonomy(below) should we add info to the title that the entity has been deleted? Now it just fallbacks to the `slug`.

Reply:
Added a title specifying the entitye refenced was deleted.